doc/api/api.md

```diff
@@ -17,6 +17,6 @@ To learn about what specific calls can be made read the [wallet foreign API doc]
 
 ### Wallet Owner API
 
-The wallet owner API is an endpoint to manage the user wallet: broadcast transaction, sign transaction, see the current balance... This REST API can be started with the `grin wallet owner_api` command and will listen on `localhost:3420`. This endpoint must **never** be exposed to the outside world.
+The wallet owner API is an endpoint to manage the user wallet: broadcast transaction, sign transaction, see the current balance... This REST API can be started with the `grin wallet owner_api` command and will listen on `localhost:13420`. This endpoint must **never** be exposed to the outside world.
```

Comment:
Seems the docs are using mainnet ports now?

Reply:
ah, this is actually an issue in grin - it should be fairly easy to fix the issue and avoid amending the docs?

Reply:
should be fixed in https://github.com/mimblewimble/grin/pull/2405